runners/direct-java/build.gradle

```diff
@@ -18,15 +18,19 @@
 
 import groovy.json.JsonOutput
 
+//the list used for defining include set and the set of project level dependencies
+def dependOnProjects = [":beam-model-pipeline", ":beam-runners-core-construction-java",
+                        ":beam-runners-core-java", ":beam-runners-local-java-core",
+                        ":beam-runners-java-fn-execution", ":beam-sdks-java-fn-execution"]
+
 apply from: project(":").file("build_rules.gradle")
 applyJavaNature(shadowClosure: DEFAULT_SHADOW_CLOSURE << {
   dependencies {
     include(dependency(library.java.protobuf_java))
     include(dependency(library.java.protobuf_java_util))
-    include(project(path: ":beam-model-pipeline", configuration: "shadow"))
-    include(project(path: ":beam-runners-core-construction-java", configuration: "shadow"))
-    include(project(path: ":beam-runners-core-java", configuration: "shadow"))
-    include(project(path: ":beam-runners-local-java-core", configuration: "shadow"))
+    dependOnProjects.each {
+      include(project(path: it, configuration: "shadow"))
+    }
   }
   relocate "org.apache.beam.runners.core", getJavaRelocatedPath("runners.core")
```

Comment:
beam-sdks-java-fn-execution is in the org.apache.beam.sdk.fn namespace and needs to be relocated
beam-runners-local-java-core is in the org.apache.beam.runners.local namespace and needs to be relocated